addons/mail/models/mail_channel.py

```diff
@@ -125,14 +152,14 @@ def _check_moderator_email(self):
         if any(not moderator.email for channel in self for moderator in channel.moderator_ids):
             raise ValidationError(_("Moderators must have an email address."))
 
-    @api.constrains('moderator_ids', 'channel_partner_ids', 'channel_last_seen_partner_ids')
+    @api.constrains('moderator_ids', 'channel_partner_ids')
     def _check_moderator_is_member(self):
```

Comment:
Should depend on `moderator_ids.partner_id` (already an issue before current PR)